tests/ctst/features/dmf.feature

```diff
@@ -73,3 +73,20 @@ Feature: DMF
     |           Non versioned |           2 |        100 |           1 |
     |               Versioned |           2 |        100 |           1 |
     |               Suspended |           2 |        100 |           1 |
+
+    @2.7.0
+    @PreMerge
+    @Dmf
+    @ColdStorage
+    Scenario Outline: Override of a cold object
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-cold" location
+    And <objectCount> objects "obj" of size <objectSize> bytes
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-cold"
+    And dmf volume should contain <objectCount> objects
+    Given <objectCount> objects "obj" of size <objectSize> bytes
+    Then dmf volume should contain 0 objects
```

Comment:
* this will probably get flaky, as the "new" obejct will be transitionned as well....and when that happens, we will have 1 object (again) in DMF
→ maybe the "safer" test would be wait again for transition, and check that indeed there is only 1 object eventually. (this would also introduce some delay, to help ensure the oplog/GC/.. messages have been processed)

* did you double check to confirm that this test actually _fails_ without your changes (i.e. the cloudserver bump)?